Makefile.pre.in

```diff
@@ -565,6 +565,8 @@ OBJECT_OBJS=	\
 		Objects/unicodectype.o \
 		Objects/unionobject.o \
 		Objects/weakrefobject.o \
+		Objects/interpolationobject.o \
+		Objects/templateobject.o \
```

Comment:
The list above is sorted, can you keep the order please?

Reply:
Now sorted